blocks/lists.js

```diff
@@ -217,10 +217,10 @@ Blockly.Blocks['lists_create_with'] = {
    * @this {Blockly.Block}
    */
   saveConnections: function(containerBlock) {
-    var itemBlock = containerBlock.getInputTargetBlock('STACK');
-    var i = 0;
+    let itemBlock = containerBlock.getInputTargetBlock('STACK');
+    let i = 0;
     while (itemBlock) {
```

Comment:
There is potential here as well to replace this with a `for` loop which might be more readable.

Reply:
Done